<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuVersionWidget</class>
 <widget class="QWidget" name="MenuVersionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>540</height>
   </rect>
  </property>
  <widget class="QLabel" name="numTitleLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>150</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>机顶盒编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelTitleLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>180</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>机顶盒型号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateTitleLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>210</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出厂时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hwTitleLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>240</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件版本:</string>
   </property>
  </widget>
  <widget class="QLabel" name="swTitleLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>270</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件版本:</string>
   </property>
  </widget>
  <widget class="QLabel" name="loaderTitleLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>300</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>loader版本:</string>
   </property>
  </widget>
  <widget class="QLabel" name="faceTiteLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>330</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>界面版本:</string>
   </property>
  </widget>
  <widget class="QLabel" name="appTitleLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>360</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>应用类型:</string>
   </property>
  </widget>
  <widget class="QLabel" name="areaTitleLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>390</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>本机分区号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="numLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>150</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>180</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>210</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hwLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>240</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="swLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>270</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loaderLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>300</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="faceLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>330</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="appLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>360</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="areaLabel">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>390</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bottomLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>470</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>【菜单】退出当前页面</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
